enterprise/frontend/src/embedding-sdk/components/public/InteractiveDashboard/InteractiveAdHocQuestion.tsx

```diff
@@ -0,0 +1,82 @@
+import { useEffect, useState } from "react";
+
+import type { SdkClickActionPluginsConfig } from "embedding-sdk";
+import { InteractiveQuestionResult } from "embedding-sdk/components/public/InteractiveQuestion/InteractiveQuestionResult";
+import { useDispatch, useSelector } from "metabase/lib/redux";
+import { initializeQBRaw } from "metabase/query_builder/actions";
+import { getQueryResults } from "metabase/query_builder/selectors";
+
+interface InteractiveAdHocQuestionProps {
+  questionPath: string;
```

Comment:
what is QuestionPath here? Does this refer to the URL of the question? (e.g. `/question/140-best-selling-products`); I think it might be a good idea to add a docstring here as it might not be immediately obvious to the SDK users on what this is.

Reply:
That makes sense!